packages/dds/sequence/README.md

```diff
@@ -1,24 +1,24 @@
 # @fluidframework/sequence
 
 The **@fluidframework/sequence** packages supports distributed data structures which are list-like.  It includes
-SharedString for storing storing and simultaneously editing a sequence of text. Note that SharedString is a sequence
+SharedString for storing and simultaneously editing a sequence of text. Note that SharedString is a sequence
 DDS but it has additional specialized features and behaviors for working with text.
 
 Sequence DDSes share a common base class, SharedSegmentSequence. For the remainder of this document, the term
 *sequence* refers to this base class.
 
-*Item*s are the individual units that are stored within the sequence (i.e. in a SharedString the items are characters),
+*Item*s are the individual units that are stored within the sequence (e.g. in a SharedString the items are characters),
 but regardless of the type of data stored in the sequence, every item in a sequence is at a specific *position* starting
 at 0, similar to an array. However, sequences differ from arrays in that the positions can move as local and remote
 editors make modifications to the sequence.
 
 As its name suggests, SharedSegmentSequence is composed of *segments*. Segments are the unit that the sequence works
-with internally, and contain items within them. Thus, every segment has a length of at least 1 -- that is, it contains
-at least one item -- and segments may be split and merged arbitrarily as the sequence is edited. This means the length
+with internally, and contain items within them. Thus, every segment has a length of at least 1 — that is, it contains
```

Comment:
Using two dashes is intentional. In rendered markdown (but maybe not on GitHub?) creates a typographic en-dash instead of a simple dash.

Reply:
Yes, github is displaying as two hyphens. :( I'll change it back.

Reply:
— can you just copy that char?

Reply:
em dash is what I inserted. The internet is telling me "--" is a Smartypants thing. I don't feel super strongly about this, but the two hyphens in the output looks weird.